src/python/nxt_python.c

```diff
@@ -448,8 +449,22 @@ nxt_python_set_target(nxt_task_t *task, nxt_python_target_t *target,
                   callable, module_name);
         goto fail;
     }
+    
+    value = nxt_conf_get_object_member(conf, &factory_flag_str, NULL);
 
-    if (nxt_slow_path(PyCallable_Check(obj) == 0)) {
+
+    if(value != NULL && nxt_conf_get_boolean(value)) {
+        if (nxt_slow_path(PyCallable_Check(obj) == 0)) {
+            nxt_alert(task, "factory \"%s\" in module \"%s\" can not be called to fetch callable", callable, module_name);
```

Comment:
We need to split this up to fit in 80 chars... Something like

```c
            nxt_alert(task,
                      "factory \"%s\" in module \"%s\" "
                      "can not be called to fetch callable",
                      callable, module_name);
```

That way we at least avoid breaking up the majority of the normal string...